<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>484</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log in</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>League</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="leagueComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sessIDLabel">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>SessionID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sessionIDLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="sessIDCheckBox">
     <property name="text">
      <string>Use SessionID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rembmeCheckBox">
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>emailLineEdit</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sessIDLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessIDCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sessionIDLineEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
